packages/theme-patternfly-org/templates/mdx.css

```diff
@@ -356,6 +356,7 @@
 }
 
 .ws-mdx-content {
+  flex-grow: 1;
```

Comment:
Now that you have this, you shouldn't need to set this width:

https://github.com/patternfly/patternfly-org/blob/77ae9a929eeddb6e2abb088eec25dc3e3f7ff65d/packages/theme-patternfly-org/templates/mdx.css#L364-L368

That element already has a `max-width`, and it's a block element, so it will grow to be 100% of it's parent up to 832px

https://github.com/patternfly/patternfly-org/blob/77ae9a929eeddb6e2abb088eec25dc3e3f7ff65d/packages/theme-patternfly-org/templates/mdx.css#L369-L371